src/Symfony/Component/Notifier/Exception/UnsupportedSchemeException.php

```diff
@@ -91,7 +91,7 @@ class UnsupportedSchemeException extends LogicException
     /**
      * @param string[] $supported
      */
-    public function __construct(Dsn $dsn, string $name = null, array $supported = [])
+    public function __construct(Dsn $dsn, string $name = null, array $supported = [], \Throwable $previous = null)
```

Comment:
Do you have a use case where this is needed?

Reply:
Forbidding to chain exceptions is really a bad anti-pattern. I would simply always allow it.

Reply:
Fair. 

I just dont know of a situation where you would like to chain a `UnsupportedSchemeException`.